core/types/calls.cc

```diff
@@ -1461,36 +1461,54 @@ class T_attached_class : public IntrinsicMethod {
     }
 } T_attached_class;
 
-class T_must : public IntrinsicMethod {
-public:
-    void apply(const GlobalState &gs, const DispatchArgs &args, DispatchResult &res) const override {
-        if (args.args.empty()) {
-            return;
+namespace {
+
+enum class MustBehavior {
+    T,
+    Magic,
+};
+
+void implementMust(const GlobalState &gs, const DispatchArgs &args, DispatchResult &res, MustBehavior behavior) {
+    if (args.args.empty()) {
+        return;
+    }
+    const auto loc = args.callLoc();
+    if (!args.args[0]->type.isFullyDefined()) {
+        if (behavior == MustBehavior::Magic) {
+            /* Nothing */
```

Comment:
If you're already committed to exhausting the elements of the `enum class`, you may as well switch this from an `if` to a `switch` so that Clang will warn for future additions to this enum.